src/js/components/message-indicator.jsx

```diff
@@ -70,6 +71,7 @@ export class MessageIndicatorComponent extends Component {
     }
 
     componentWillReceiveProps({unreadCount}) {
+        observable.trigger('unreadCount', unreadCount);
```

Comment:
This should be triggered here https://github.com/smooch/smooch-js/blob/integration/src/js/smooch.jsx#L102 instead right after `handleNotificationSound();`.